src/lib/projects/YearlyEvents.svelte

```diff
@@ -19,18 +47,26 @@
 			<label class="ml-4" for="only-members">Nur Mitglieder</label>
 		</div>
 	</div>
-	<section class="mt-12">
-		<img src={EventImage} alt="" class="mt-12 rounded-t-2xl" />
-		<div class="p-6 rounded-b-2xl bg-soft-yellow flex">
-			<div class="flex flex-col justify-center items-center">
-				<h1 class="text-5xl font-serif font-bold">24</h1>
-				<h1>August</h1>
+	{#each events as event}
+		<section class="mt-12">
+			<img src={event.image} alt="" class="mt-12 rounded-t-2xl" />
+			<div
+				class="p-6 rounded-b-2xl flex shadow-lg {event.for_all
+					? 'bg-soft-yellow'
+					: 'bg-light-green'}"
+			>
+				<div class="mt-4 flex flex-col items-center">
+					<h1 class="text-5xl font-serif font-bold">{event.day}</h1>
```

Comment:
Just a note: I'm not sure if we should count the day as a heading, but maybe it makes sense?

Reply:
h1 around day and month changed to div. Maybe we will find a more fitting tag?